homeassistant/components/homekit/type_humidifiers.py

```diff
@@ -229,19 +236,7 @@ def _set_chars(self, char_values: dict[str, Any]) -> None:
         if self._target_humidity_char_name in char_values:
             state = self.hass.states.get(self.entity_id)
             assert state
-            max_humidity = state.attributes.get(ATTR_MAX_HUMIDITY, DEFAULT_MAX_HUMIDITY)
-            max_humidity = round(max_humidity)
-            max_humidity = min(max_humidity, 100)
-
-            min_humidity = state.attributes.get(ATTR_MIN_HUMIDITY, DEFAULT_MIN_HUMIDITY)
-            min_humidity = round(min_humidity)
-            min_humidity = max(min_humidity, 0)
-            # The min/max humidity values here should be clamped to the HomeKit
-            # min/max that was set when the accessory was added to HomeKit so
-            # that the user cannot set a value outside of the range that was
-            # originally set as it could cause HomeKit to report the accessory
-            # as not responding.
-
```

Comment:
This doesn't matter anymore since pyhap will clamp them anyways, and if the underlying value changes we now rebuild the accessory anyways so it never gets here.